firmware/applications/SignBuddy/tsc.c

```diff
@@ -4,19 +4,89 @@
 #include "common.h"
 #include "logger.h"
 
-#define CALIBRATION_SAMPLES    8
-#define TOUCH_TRIGGER_LEVEL    50
+#include <string.h>
+
+#define CALIBRATION_SAMPLES            8
+#define TOUCH_TRIGGER_LEVEL            50
+
+#define CHANNEL0                       0
+#define CHANNEL1                       1
+
+#define TOUCH_SENSOR_0                 TSC_GROUP3_CHANNEL0
+#define TOUCH_SENSOR_1                 TSC_GROUP4_CHANNEL0
+#define TOUCH_SENSOR_2                 TSC_GROUP5_CHANNEL0
+#define TOUCH_SENSOR_3                 TSC_GROUP6_CHANNEL0
+#define TOUCH_SENSOR_4                 TSC_GROUP7_CHANNEL0
+#define TOUCH_SENSOR_5                 TSC_GROUP8_CHANNEL0
```

Comment:
I think we should move these to the board header and rename like  TOUCH_0_CHANNEL

Reply:
these were temporary - it is now what I wanted them to be